_includes/portfolio.html

```diff
@@ -13,6 +13,13 @@
 {% else %}
 	{% assign button = "Launch Site" %}
 {% endif %}
+{% if include.defaultThumbnail %}
+	{% assign defaultThumbnail = include.defaultThumbnail %}
+{% elsif page.portfolio.defaultThumbnail %}
+	{% assign defaultThumbnail = page.portfolio.defaultThumbnail %}
+{% else %}
+	{% assign defaultThumbnail = "/style/images/docs/defaultThumbnail.png" %}
```

Comment:
I think this should be in `/style/images/`, the portfolio isn't necessarily limited to docs.